packages/access-mom6/package.py

```diff
@@ -36,7 +39,7 @@ class AccessMom6(CMakePackage):
     depends_on("fms@2023.02: precision=64 +large_file ~gfs_phys ~quad_precision")
     depends_on("fms +openmp", when="+openmp")
```

Comment:
```suggestion
    depends_on("fms +openmp", when="+openmp")
    depends_on("fms@2025.02:", when="+openmp~access3")
```

Sorry this is another hidden dep I forgot about - see https://github.com/ACCESS-NRI/ACCESS-OM3/pull/100#issuecomment-2893858219

Reply:
How come it's not needed with `+access3`?

Reply:
Apparently it is - https://github.com/ACCESS-NRI/ACCESS-OM3/pull/102#issuecomment-2896046821

It's related to the CMake exports again ...

I added OpenMP C Component to the FMS exports in https://github.com/NOAA-GFDL/FMS/pull/1663

A fixed suggestion:

```suggestion
    depends_on("fms@2025.02: +openmp", when="+openmp")
```

(Also if relying on new fms is problematic, we could patch the fms spack package ? The dependency is only related to cmake exports)

Reply:
I think it's fine to rely on new FMS. Its good to keep our dependencies up to date anyway.

The only annoying thing is that 2025.02 isn't in the FMS spack package yet, but I think [this](https://github.com/ACCESS-NRI/ACCESS-OM3/blob/0cfa1823f1d591bece55f8cb6903831b6b06c352/spack.yaml#L53) is a fine approach for now.